ansible/roles/test/tasks/qos_sai.yml

```diff
@@ -37,14 +37,20 @@
         - lldpd
         - lldp-syncd
 
-    - name: Disable bgpd
+    - name: Ensure BGP Daemon stopped
       become: yes
-      lineinfile: dest=/etc/quagga/daemons
-                  regexp=^bgpd=.*$
-                  line='bgpd=no'
-      notify:
-        - Restart Quagga Daemon
+      supervisorctl: state=stopped name={{item}}
       delegate_to: "{{ ansible_host }}_bgp"
+      with_items:
+        - bgpd
```

Comment:
Why do you need a loop? you can specify bgpd directly in 'supervisorctl' command

Reply:
@neethajohn Updated.